src/components/infoBox/InfoBox.tsx

```diff
@@ -1,11 +1,15 @@
+import { Link } from "react-router-dom";
+
 import * as S from "./InfoBox.styled";
 
 export const InfoBox = () => {
 	return (
 		<S.InfoBoxStyledContainer>
 			<h3>Der ganze Netflix-Spaß schon ab 4,99 €.</h3>
 			<p>Netflix mit ein paar Werbeunterbrechungen zum Sparpreis genießen.</p>
-			<S.Button>Weitere Infos</S.Button>
+			<S.Button>
+				<Link to="/subscription">Weitere Infos</Link>
+			</S.Button>
```

Comment:
Zawsze możesz zrobić dedykowany komponent Button, globalny, który będzie przyjmował variant, text oraz link, jeżeli link będzie, to renderujesz <Link> jezeli nie to <Button> po prostu (wewnatrz tego reuzywalnego komponentu)